test/files/pos/cycle-jsoup/Jsoup_1.java

```diff
@@ -0,0 +1,24 @@
+package org.jsoup;
+
+import java.net.URL;
+
+public class Jsoup_1 {
+  public static Document parse(URL url, int timeoutMillis)  {
+    return null;
+  }
+
+  public static class Document extends Element {
+    public class OutputSettings {
+    }
+  }
+
+  public static class Element extends Node {
+    void outerHtmlTail(StringBuilder accum, int depth, Document.OutputSettings out) {
+    }
+  }
+
+  public static class Node {
+    protected void indent(StringBuilder accum, int depth, Document.OutputSettings out) {
+    }
+  }
+}
```

Comment:
I checked that this minimization fails on Scala 2.12.8 without `-Ybreak-cycles`.